lib/ranked-model/ranker.rb

```diff
@@ -69,6 +69,16 @@ def update_rank! value
           update_all(ranker.column => value)
       end
 
+      def rank_after(record)
```

Comment:
Technically the variable should be `ranked_record` to record the fact we're dealing with a ranker here not the actual record. It could also even be `rank` so `def rank_after(rank)`.